docs/notes/2.23.x.md

```diff
@@ -45,6 +45,8 @@ Support for parametrizing grouped parametrizations. (e.g. `**parametrize(resolve
 
 Docker inference is improved. Pants can now make inferences by target address for targets supporting `pants package`, and `file` targets can be included by filename. See the [documentation on Docker dependency inference](https://www.pantsbuild.org/2.23/docs/docker#dependency-inference-support) for details
 
+Experimental support for a rust-based dockerfile parser can be enabled via `[dockerfile-parser].use_rust_parser` option.
```

Comment:
This is called experimental here, but there's no indication of that status "closer" to the option. Can you maybe adjust the doc string of the option to re-iterate that status too?

Reply:
Sure!